scripts/py_matter_yamltests/matter_yamltests/pics_checker.py

```diff
@@ -52,7 +52,9 @@ def check(self, pics) -> bool:
 
         self.__expression_index = 0
         tokens = self.__tokenize(pics)
-        return self.__evaluate_expression(tokens, self.__pics)
+        if self.__pics is not None:
+            return self.__evaluate_expression(tokens, self.__pics)
+        return True
```

Comment:
Pretty sure this should actually be false

Reply:
I have done a PR with a test for the case where no pics file is used.